src/remote/blobs.go

```diff
@@ -311,6 +311,9 @@ func (c *Client) downloadBlobs(f func(ch chan<- *blob) error) error {
 
 // retrieveByteStream receives a file back from the server as a byte stream.
 func (c *Client) retrieveByteStream(b *blob) error {
+	if b.Digest == nil {
+		return fmt.Errorf("can't retrieve byte stream from nil digest")
```

Comment:
ooh dear does this happen? it probably shouldn't...

Reply:
Maybe - it's to do with reading [these digest fields](https://github.com/bazelbuild/remote-apis/blob/master/build/bazel/remote/execution/v2/remote_execution.proto#L870-L885) which I guess are optional, strictly speaking.